stencil-library/src/components.d.ts

```diff
@@ -15,7 +15,7 @@ export namespace Components {
         "styleOverrides"?: string;
         "tokenize": (clientId: string, paymentMethodMetadata: any, account?: string) => Promise<any>;
         "validate": () => Promise<any>;
-        "validationStrategy": 'onChange' | 'onBlur' | 'onSubmit' | 'onTouched' | 'all';
+        "validationMode": 'onChange' | 'onBlur' | 'onSubmit' | 'onTouched' | 'all';
```

Comment:
Since we're changing the public API here, this should actually be considered a "breaking change" (even though this feature was broken before) so the version number should go to `1.0.0`